integration-test/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "netlify build",
```

Comment:
can you change this to use the `netlify-build` binary, like Mickael suggested? don't forget to install the package explicitly